libdnf/rpm/package_set_iterator.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "libdnf/rpm/package_set_iterator.hpp"
+
+#include "libdnf/rpm/package_set.hpp"
+#include "libdnf/rpm/package_set_impl.hpp"
+#include "libdnf/rpm/package_set_iterator_impl.hpp"
+
+
+namespace libdnf::rpm {
+
+
+PackageSetIterator::PackageSetIterator(const PackageSet & package_set) : pImpl{new Impl(package_set)} {
+    package.id = *(*pImpl);
```

Comment:
Name of the value is not bad, but could be better. It represent an iterator value, therefore it could be named according to it. But this is only a minor note and can be closed as resolved directly.

Reply:
Changed to current_value